docs/pow.md

```diff
@@ -0,0 +1,71 @@
+# Proof-of-work in the faucet
+
+The faucet uses POW challenges for anti-dos and dynamically shifts the
```

Comment:
First time you mention an acronym you give the full definition. Then, you can use it as much as you want.

```suggestion
The faucet uses Proof-of-Work (POW) challenges for anti-dos and dynamically shifts the
```